pkg/staticnetworkconfig/generator.go

```diff
@@ -295,3 +296,29 @@ func (s *StaticNetworkConfigGenerator) formatMacInterfaceMap(macInterfaceMap mod
 	sort.Strings(lines)
 	return strings.Join(lines, "\n")
 }
+
+func GenerateStaticNetworkConfigArchive(files []StaticNetworkConfigData, infraEnvID string) (*bytes.Buffer, error) {
+	buffer := new(bytes.Buffer)
+	w := tar.NewWriter(buffer)
+	for _, file := range files {
+		path := filepath.Join("/network", infraEnvID, file.FilePath)
```

Comment:
Why are we using this particular path?
I feel like it would be better to just keep the path the files would have on the host.

Reply:
Gotcha, I'll revert it back!

My initial thought process was to contain the files in a folder such as `network-files` so that ways the user can easily find them on their local filesystem when they untar the archive. 

Although, I can see how having it be the same path as it exists on the host(s) makes sense